src/ng-package.schema.json

```diff
@@ -77,5 +77,40 @@
     }
   },
   "required": [],
-  "additionalProperties": false
+  "additionalProperties": false,
+  "definitions": {
+    "assetPattern": {
+      "oneOf": [
+        {
+          "type": "object",
```

Comment:
Nit: can you also add

```
"followSymlinks": {
  "type": "boolean",
  "default": false,
  "description": "Allow glob patterns to follow symlink directories. This allows subdirectories of the symlink to be searched."
},
```

Reply:
Looks like you missed the default value.

Reply:
It throws `strict mode: default is ignored for: data7.followSymlinks` when testing, so I removed the it. I'll try to solve it.

Reply:
Solved